keps/0001-kubernetes-enhancement-proposal-process.md

```diff
@@ -145,7 +145,7 @@ These KEPs will be owned by SIG-architecture and should be seen as a way to comm
 ### KEP Template
 
 <b>
-The template for a KEP is precisely defined [here](/keps/NNNN-kep-template).
+The template for a KEP is precisely defined <a href="/keps/NNNN-kep-template">here</a>.
 </b>
```

Comment:
To be honest, the `<b>` should probably be removed here and just use markdown bold instead. 😬

Reply:
Makes sense. Updated.

Reply:
Thanks!